tests/e2e-kfddistribution-upgrades.sh

```diff
@@ -10,8 +10,8 @@ echo "Executing furyctl for the initial setup 1.29.4"
 /tmp/furyctl apply --config tests/e2e/kfddistribution-upgrades/furyctl-init-cluster-1.29.4.yaml --outdir "$PWD" --disable-analytics
 
 echo "----------------------------------------------------------------------------"
-echo "Executing upgrade to 1.30.0"
+echo "Executing upgrade to 1.30.1"
 # we set the switch date for Loki to "tomorrow". Notice that `-d flag` does not work on Darwin, you need to use `-v +1d` instead.
-# this is needed only when upgrading from 1.29.4 to 1.30.0 (and equivalent versions)
-yq -i ".spec.distribution.modules.logging.loki.tsdbStartDate=\"$(date -I -d '+1 day')\"" tests/e2e/kfddistribution-upgrades/furyctl-init-cluster-1.30.0.yaml
-/tmp/furyctl apply --upgrade --config tests/e2e/kfddistribution-upgrades/furyctl-init-cluster-1.30.0.yaml --outdir "$PWD" --distro-location ./ --force upgrades --disable-analytics
+# this is needed only when upgrading from 1.29.4 to 1.30.1 (and equivalent versions)
+yq -i ".spec.distribution.modules.logging.loki.tsdbStartDate=\"$(date -I -d '+1 day')\"" tests/e2e/kfddistribution-upgrades/furyctl-init-cluster-1.30.1.yaml
+/tmp/furyctl apply --upgrade --config tests/e2e/kfddistribution-upgrades/furyctl-init-cluster-1.30.1.yaml --outdir "$PWD" --distro-location ./ --force upgrades --disable-analytics
```

Comment:
there's no upgrade path from 1.29.4 to 1.30.1, but this works because we are setting the `--force upgrades` flag and there are no major changes between 1.30.0  and 1.30.1, maybe we could add a comment saying that for future reference.

Reply:
The upgrade on that script is done from 1.30 to 1.30.1, there is no 1.29.4, i will remove the comment